pkg/hs/urbit-king/lib/Urbit/King/Main.hs

```diff
@@ -541,30 +547,31 @@ newShip CLI.New{..} opts = do
           Nothing -> error "Urbit.ob didn't produce string with ~"
           Just x  -> pure x
 
-    bootFromSeed :: Pill -> Seed -> RIO HostEnv ()
-    bootFromSeed pill seed = do
-      ethReturn <- dawnVent nEthNode seed
+    bootFromSeed :: Pill -> Feed -> RIO HostEnv ()
```

Comment:
Arguably we should rename this `bootFromFeed`. Although may be moot with urth/mars